src/ae_epoll.c

```diff
@@ -87,10 +87,12 @@ static int aeApiAddEvent(aeEventLoop *eventLoop, int fd, int mask) {
     return 0;
 }
 
-static void aeApiDelEvent(aeEventLoop *eventLoop, int fd, int delmask) {
+static void aeApiDelEvent(aeEventLoop *eventLoop, int fd, int mask) {
```

Comment:
We've sanitized the `mask` and we can now use `fe->mask` directly, which makes the `mask` argument seem redundant. We may want to update this function signature and get rid of this argument?

Reply:
Based on the current implementation, the `kqueue` impl still needs the `mask` parameter to know which filters should be deleted.  
I'm afraid that, since the `aeApiXxx` functions are used as something like "abstract methods", the `epoll` impl has to follow the method signature, even if it's not used.

Reply:
@panjf2000 hi, are there more suggestions about the code, or shall we resolve the conversation?